### PR TITLE
Helm Chart updates

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: GitHub <noreply@github.com>

### CODE REVIEW

File: helm-chart/templates/nextcloud-aio-database-deployment.yaml [hunk 1]
```diff
@@ -21,14 +21,13 @@ spec:
         io.kompose.network/nextcloud-aio: "true"
         io.kompose.service: nextcloud-aio-database
     spec:
-      initContainers:
```
Comment (st3iny) [inline helm-chart/templates/nextcloud-aio-database-deployment.yaml:24]: This line needs to stay.

**Review by st3iny — CHANGES_REQUESTED** [review]

File: helm-chart/templates/nextcloud-aio-database-deployment.yaml (hunk 1 above)
Comment (szaimen) [reply helm-chart/templates/nextcloud-aio-database-deployment.yaml]: right, removed by mistake

**Review by st3iny — COMMENTED** [review]
Services look fine and work. However, the nextcloud deployment is not working correctly.

**Comment by szaimen** [comment]
> owever, the nextcloud deployment is not working correctly.

lets fix this in a follow-up if it should be reproducible.

**Pull request merged**